ws-nextjs-app/pages/[service]/new_live/[id]/Post/styles.tsx

```diff
@@ -18,14 +18,14 @@ export default {
       alignItems: 'flex-start',
       display: 'flex',
       flexWrap: 'wrap',
-      boxShadow: `inset 0 ${pixelsToRem(2)}rem ${palette.BRAND_BACKGROUND}`,
+      borderTop: `solid ${pixelsToRem(2)}rem ${palette.BRAND_BACKGROUND}`,
```

Comment:
I want to check I'm right to change this? I wonder if we were using Box Shadow because of where the transparent border was. I have changed the transparent border so now wraps the full comppnent (including the breaking news banner), so I figure it's fine to use borderTop here now. The borderTop seems to work better when the page is resized.

Reply:
I'm not clear on the rationale to initially use `boxShadow`. Given 
`border-top` is used in the spec in the designs under the `Design` page,
I'm happy to use `borderTop` here.
![Screenshot 2023-09-22 at 21 53 24 (3)](https://github.com/bbc/simorgh/assets/20963836/3c14ad13-a662-40b9-9330-d497d5cd0397)

Reply:
From adding this, when you view in Firefox with the colour preferences changed so you can see all the transparent borders we have aswell, there is now the apperance of a bigger border at the top of the post, unless I'm mistaken if we use box shadow for this one, then the 'shadow' will not show up in this mode, and the transparent border will still appear to be the same dimenions on all sides? Ehich is what we want, though I note your comment about resizing the page @Isabella-Mitchell. Alternatively we can keep this as a border and compensate for this in the transparent border dimenensions...

Reply:
Futher to this, UX should be able to advise if they expect a 'border' in this mode below the timestamp to provide an affordance for the red background...

<img width="1057" alt="Screenshot 2023-09-25 at 14 26 46" src="https://github.com/bbc/simorgh/assets/1061401/502a8567-376a-4f6b-8212-c19bfd5898f2">